src/users/containers/invite-user/invite-user.component.ts

```diff
@@ -220,8 +220,8 @@ export class InviteUserComponent implements OnInit, OnDestroy {
         firstName: [(this.f.firstName.errors && this.f.firstName.errors.required)],
         lastName: [(this.f.lastName.errors && this.f.lastName.errors.required)],
         email: [
-          (this.f.email.errors && this.f.email.errors.required),
-          (this.f.email.errors && this.f.email.errors.email),
+          (this.f.email.errors && this.f.email.errors.required) ||
+          (this.f.email.errors && this.f.email.errors.email)
         ],
         roles: [(this.f.roles.errors && this.f.roles.errors.requireOneCheckboxToBeChecked)],
```

Comment:
I don't know how this formValidationData data structure is used but I'm interested in the original code it put the single undefined/boolean value inside an array [] for these firstName/lastName/roles value, and after the email one changed to OR condition would it be more suitable just return the value(undefined/boolean) rather than in an array.

Reply:
It was basically adding duplicate hidden accessibility validation error messages to the page, one of which was undefined if the user did not enter an email address. I can show you this in action on a preview URL if you like.